CONTRIBUTING.md

```diff
@@ -3,44 +3,31 @@
 Contributions are welcome, and they are greatly appreciated! Every little bit
 helps, and credit will always be given.
 
-You can contribute in many ways:
+- Report bugs, request features or submit feedback as a [GitHub Issue](https://github.com/snowexhackweek/website/issues).
```

Comment:
change link to generic github 'issues' docs?

Reply:
or change `snowexhackweek` --> TEMPLATE. If we want to get fancy eventually, we could have a script that renames every occurance of TEMPLATE with the new repository name https://github.com/stactools-packages/template/blob/main/scripts/rename documented here https://github.com/stactools-packages/template#how-to-use

Reply:
Yes to both (make generic for now, ultimately try to get fancier).